### PR TITLE
More improvements on TACA running Anglerfish

### PR DESCRIPTION
**Pull request by kedhammar**
1. Use abspath for Anglerfish stderr path
2. Make it possible to instantiate ONT run w/o specifying the type
3. Add more info to the ONT db update subcommand

### CODE REVIEW

File: taca/nanopore/ONT_run_classes.py [hunk 1]
```diff
@@ -65,9 +65,10 @@ def __init__(self, run_abspath: str):
                 self.rsync_options[k] = None
 
         # Get transfer details, depending on run type and instrument
-        self.transfer_details = CONFIG["nanopore_analysis"]["run_types"][self.run_type][
-            "instruments"
-        ][self.instrument]
+        if hasattr(self, "run_type"):
+            self.transfer_details = CONFIG["nanopore_analysis"]["run_types"][self.run_type][
```
Comment (alneberg) [inline taca/nanopore/ONT_run_classes.py:69]: Does this need a default value as well? If `run_type` is not available?

Comment (kedhammar) [reply taca/nanopore/ONT_run_classes.py:69]: The run_type attribute should always be available if it's used, so I'm content with it throwing an error if that is ever the case

File: taca/nanopore/ONT_run_classes.py [hunk 2]
```diff
@@ -68,9 +68,10 @@ def __init__(self, run_abspath: str):
                 self.rsync_options[k] = None
 
         # Get transfer details, depending on run type and instrument
-        self.transfer_details = CONFIG["nanopore_analysis"]["run_types"][self.run_type][
-            "instruments"
-        ][self.instrument]
+        if hasattr(self, "run_type"):
```
Comment (alneberg) [inline taca/nanopore/ONT_run_classes.py:71]: I would change this to set `self.run_type = None  # Will be set in subclasses` and `if self.run_type is not None`. I think it's easier to understand. What do you think?

Comment (kedhammar) [reply taca/nanopore/ONT_run_classes.py]: [fe0f44e](https://github.com/SciLifeLab/TACA/pull/410/commits/fe0f44e50509e8a78d80f3d162dae9026b669083)

**Review by alneberg — APPROVED** [review]
Thank you!

**Pull request merged**